hathor/indexes/manager.py

```diff
@@ -165,6 +171,8 @@ def __init__(self, db: 'rocksdb.DB') -> None:
         self.tokens = None
         self.height = RocksDBHeightIndex(self._db)
         self.mempool_tips = RocksDBMempoolTipsIndex(self._db)
+        # self.deps = RocksDBDepsIndex(self._db)
```

Comment:
Should we use this one?

Reply:
Yes. I've reversed this now. I had it because I was only testing MemoryDepsIndex while I finished the interface refactor.